File: labs/solidworks/swex/pmpage/controls/selection-box/SelectionBox.Single.cs
public class SelectionBoxDataModel
{
    [SelectionBox(swSelectType_e.swSelSOLIDBODIES)]
    public IBody2 Body { get; set; }

    [SelectionBox(swSelectType_e.swSelEDGES, swSelectType_e.swSelNOTES, swSelectType_e.swSelCOORDSYS)]
    public object Dispatch { get; set; }
}

File: labs/solidworks/swex/pmpage/controls/selection-box/SelectionBox.Single.vb
Public Class SelectionBoxDataModel

    <SelectionBox(swSelectType_e.swSelSOLIDBODIES)>
    Public Property Body As IBody2

    <SelectionBox(swSelectType_e.swSelEDGES, swSelectType_e.swSelNOTES, swSelectType_e.swSelCOORDSYS)>
    Public Property Dispatch As Object

End Class